build.gradle

```diff
@@ -438,6 +442,77 @@ project(':iceberg-aws') {
   }
 }
 
+project(':iceberg-delta-lake') {
+  // use integration test since we can take advantages of spark 3.3 to read datafiles of delta lake table
+  // and create some tests involving sql query.
+  configurations {
+    integrationImplementation.extendsFrom testImplementation
+    integrationRuntime.extendsFrom testRuntimeOnly
+  }
+
+  dependencies {
+    implementation project(path: ':iceberg-bundled-guava', configuration: 'shadow')
+    api project(':iceberg-api')
+    implementation project(':iceberg-common')
+    implementation project(':iceberg-core')
+    implementation project(':iceberg-parquet')
+    implementation project(':iceberg-orc')
```

Comment:
I don't think delta supports ORC, this might be an unnecessary dependency.

Reply:
Just saw the comment below, so feel free to ignore, but I'd probably lean towards waiting until they add support for other file formats just so that we can add appropriate tests and make sure all the conversions/stats are handled correctly.

Reply:
Thank you for pointing yes. I re-consider the implementation and agree that it would be better to add ORC support after Delta Lake does. I removed support for getting metrics from avro and orc.

Reply:
good point about testing coverage, +1 for removing ORC and Avro in that case, it will make the codebase a lot simpler as well.